### ISSUE TITLE
Text: Improve background frame alignment and styling options

### ISSUE THREAD

**Issue by riquich** [ScottPlot/ScottPlot#1417]
## Bug Report

**Issue:** When using background frame for text plots, the alignment rule is not respected for the background frame

**Reproducing:** Plotting a text with background enabled and alignment is set anything other than Alignment.UpperLeft will reproduce the problem. If I recall correctly this issue was not present in 4.0.*

```cs
WpfPlot Root; /*XAML element*/
Text MouseLocationText;
VLine MouseTrackerVertLine;
HLine MouseTrackerHorzLine;
...
...
MouseTrackerVertLine = Root.Plot.AddVerticalLine(DateTime.Now.ToOADate(), color: Color.Red, style: LineStyle.Dash);
MouseTrackerHorzLine = Root.Plot.AddHorizontalLine(4, color: Color.Red, style: LineStyle.Dash);
MouseLocationText = Root.Plot.AddText("", 50, 150, color: Color.Red, size: 18);
MouseLocationText.Alignment = Alignment.UpperRight;
MouseLocationText.BackgroundFill = true;
MouseLocationText.BackgroundColor = Color.Black;
```

![a](https://user-images.githubusercontent.com/34640902/139676783-878b51ad-1451-43fe-8bff-982c61714244.PNG)

### System Details
* **ScottPlot Version:** 4.1.27
* **Operating System:** Windows 10
* **Application Type:** WPF
* **.NET Version:** .Net core 3.1

**Comment by AndXaf** [comment]
Dear community,

I would like to contribue - Please see attached file which contains

- corrections for above issue
- additional public properties 'BorderWidth' and 'BorderColor' which are also respected

[Text.zip](https://github.com/ScottPlot/ScottPlot/files/7575388/Text.zip)

**Comment by MachineFossil** [comment]
I would find it helpful to have BorderWidth and BorderColor properties like @AndXaf suggested.

**Comment by swharden** [comment]
I think all these issues/features are now resolved/added when I merge #1516 in a few minutes 🚀 

![image](https://user-images.githubusercontent.com/4165489/147801720-7ed3fe92-9638-49f5-9f79-03c02c7bb9ac.png)
